parachains/integration-tests/emulated/assets/asset-hub-westend/src/tests/transact.rs

```diff
@@ -0,0 +1,58 @@
+use crate::*;
+
+#[test]
+fn transact_sudo_from_relay_to_assets_para() {
+	// Init tests variables
+	// Call to be executed in Assets Parachain
+	const ASSET_ID: u32 = 1;
+
+	let call = <AssetHubWestend as Para>::RuntimeCall::Assets(pallet_assets::Call::<
+		<AssetHubWestend as Para>::Runtime,
+		Instance1,
+	>::force_create {
+		id: ASSET_ID.into(),
+		is_sufficient: true,
+		min_balance: 1000,
+		owner: AssetHubWestendSender::get().into(),
+	})
+	.encode()
+	.into();
+
+	// XcmPallet send arguments
+	let sudo_origin = <Westend as Relay>::RuntimeOrigin::root();
+	let assets_para_destination: VersionedMultiLocation =
+		Westend::child_location_of(AssetHubWestend::para_id()).into();
+
+	let weight_limit = WeightLimit::Unlimited;
+	let require_weight_at_most = Weight::from_parts(1000000000, 200000);
```

Comment:
This can be a follow up project (maybe good for someone new), but we should have some core set of "user stories" that other chains would use transact for, and then set the weight values in these tests as `actual * some_factor_of_safety`, and if the tests fail in a new release because of weight, then we need to mark it as a breaking change. That way people can rely on these weights in tests to have some future proof-ness in their runtimes.